app/controllers/resumes_controller.rb

```diff
@@ -1,8 +1,8 @@
 class ResumesController < ApplicationController
 
-  before_action :authenticate, except: [ :index ],  unless: -> { Rails.env.test? }
-  before_action :authorize, except: [:index, :update_specific_resume], unless: -> { Rails.env.test? }
-  before_action :authorize_based_on_resume_id, only: [:update_specific_resume], unless: -> { Rails.env.test? }
```

Comment:
this is not the correct way to test authentication :) used stub requests instead.

Reply:
I meant to say that the correct way is implemented in this PR